data/human/human missions.txt

```diff
@@ -4759,3 +4759,184 @@ mission "Dangerous Games 2"
+			`	His body relaxes, and he's gone.`
+			choice
+				`	(Let Riley be.)`
+					goto let
+				`	(Stay comfortingly close.)`
+					goto close
+				`	(Hold Riley.)`
+			`	You hold Riley as she silently weeps. Time goes by, but neither of you pay it any mind.`
+			label close
+			`	You take a seat, feeling the emotion of the moment wash over you as Riley silently weeps. Minutes go by, unnoticed.`
+				goto voice
+			label let
+			`	You stay at a distance as you allow Riley to process the death of her father. She silently weeps as time passes without either of you noticing.`
+			label voice
+			`	Without warning, you hear Church's voice from a corner of the room. It's coming from some sort of computer system, held aloft by hundreds of cables dangling from the ceiling.`
+			`	"It... worked." His voice fizzles, and his tone is not the one you recognize from before. Riley's head whips around, and you reflexively move your hand toward your holstered gun. "Do not be alarmed. I have been working on a personal project for over a decade now. My mind was uploaded to the modified Core you are looking at. I had to say goodbye in case the procedure failed - I did not know if it would work. You cannot upload a mind while it's still alive. I tried that many times." The tone of his voice sounds almost sorrowful, but still retains a robotic quality.`
+			choice
+				`	(Stay silent.)"`
+					goto continue
+				`	"Is this... legal?"`
+			`	"No," he replies. "Not under Republic law. I assure you that my only intention is to continue my work: nothing more."`
+			`	The authorities would probably release a substantial reward if you informed them of this.`
+			label continue
+			`	"Riley, what do you think?" you ask.`
+			`	"Dad, is that really you? How do you feel?"`
+			`	She steps closer to the Core. The AI pauses. "Riley, what do you think?" it repeats. The voice coming from the machine is oddly flat in affect. "I feel fine. Do you feel fine?" They talk for a few tentative minutes. You soon begin to realize that despite the AI's advanced technology, what Riley is talking to is but a shallow reflection of her father.`
+			`	Finally, she turns to you. "I'll admit it's not perfect, but... how many people get a second chance to spend time with their father before he's gone? Thank you for bringing me out here, and for coming. I'll never forget your part in this. For now, I'd like to stay here. You go - I'll find my own way back." She seems genuinely happy, despite the AI being less-than-ideal.`
+			choice
+				`	"Are you sure? There's enough room for all of us on my ship. You can stay for as long as you want."`
+					goto suggest
+				`	(Say goodbye.)`
+					goto goodbye
+				`	"Riley, this isn't your father. You need to move on from his death."`
+					goto death
+				`	(Turn the Core in for a bounty.)`
+					goto bounty
+				`	(Draw your gun and destroy the Core).`
```

Comment:
Maybe put this behind the choice of telling Riley that it's an AI, so that attempting to destroy the core doesn't seem as sudden.

Reply:
I'd like to keep this as an immediate reaction, in case the player is not shooting the AI because they think Riley needs to move on, but is shooting the AI because they're point-blank opposed to mind-uploading.